native/tools/native-locales

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
```

Comment:
Having a lot of small scripts can be a bit complicated.
Should we have maybe one script which generates the bulid config and generates the locales?

Something like: `generate-configs` or similar.

Reply:
What would the benefit be? I.e. why do you think a big script is better than smaller ones?

Reply:
Imo small scripts with only one purpose are better (at least as long as there are only a small amount of scripts).

Reply:
If you have only one script then this script can be self-documenting. If there is only one script to execute then you get the documentation when you run it.

```console
$ ./integreat-setup build-config generate
$ ./integreat-setup build-config json
$ ./integreat-setup build-config xcconfig
$ ./integreat-setup locales generate-native
$ ./integreat-setup circleci trigger
```

The help documents what each command does. So it is not needed to write extensive documentation.

 The commander cli also does not allow to have a script without a subcommand afaik. This means it is not possible to do something like `./do-something --verboose`. If you use commander then you have to do  `./do-something come-command --verboose`

I think that would make it clearar

Reply:
Okay, sounds reasonable :+1: Would it be okay to do this the next time we need a script? For now, I find it clear enough what our scripts are doing. Also, we don't need to call these scripts manually, they are always called automatically :)

Reply:
Sure, this is probably a separate task: https://issues.tuerantuer.org/browse/IGAPP-310

Reply:
Thanks :+1: